doc/rtd/reference/custom_modules.rst

```diff
@@ -0,0 +1,17 @@
+Custom Modules
+**************
+
+Our reference section contains support information for ``cloud-init``.
```

Comment:
Looks like this was copied from `doc/rtd/reference/index.rst`. Could we instead write something that describes something about the sub-pages? You can probably grab some text from https://github.com/canonical/cloud-init/issues/4649, but at least some description of what these sections have in common and why a user might turn to customizing cloud-init's behavior would be desirable.

Reply:
Fixed.